### ISSUE TITLE
Specifying IPython Notebook port

### ISSUE THREAD

**Issue by jairideout** [jtriley/StarCluster#191]
While teaching a workshop using the ipcluster plugin with the notebook enabled, we were unable to connect to the notebook server due to the default port of 8888 (there were no sys admins available to open the port either).

Would having a way to specify the notebook server port be something that others would use? I can find some time to work on this feature, but wanted to run it by the development group to get feedback first (and to see if this would even be accepted as an addition to the plugin).

This is related to issue #185.

**Comment by jtriley** [comment]
Yes, this would be great to have in addition to #185. One issue though is that currently the notebooks run as the "cluster_user" so only ports >1024 can be used without having to run the notebook as root.
